modules/rabbitmq/build.gradle

```diff
@@ -0,0 +1,8 @@
+description = "TestContainers :: RabbitMQ"
+
+dependencies {
+    compile project(":testcontainers")
+    implementation 'org.json:json:20180813'
```

Comment:
You can use Jackson from the core, to keep the number of dependencies as small as possible

Reply:
Sure, let me see if I can replace it with jackson